lib/stripe.rb

```diff
@@ -139,8 +139,8 @@ def self.request(method, url, api_key, params={}, headers={}, api_base_url=nil)
         raise
       end
     rescue RestClient::ExceptionWithResponse => e
-      if rcode = e.http_code and rbody = e.http_body
-        handle_api_error(rcode, rbody)
+      if response = e.response
```

Comment:
nit: at this point, i'd just kill the intermediate variable and use `e.response` directly here and in the line below